### DISCUSSION TITLE
Google ocr with email

### DISCUSSION THREAD

**Discussion by Caravanr54545** [xulihang/ImageTrans-docs#354] [Ideas]
Hi
I want use google ocr (because this BEST ocr), but ths need api. Api need cretid card, and not full free. i no want added cretid card. 
No possible make google ocr  with email? 

Thanks

**Comment by xulihang** [comment]
I have to provide a paid service for this.

The service can do the following:

1. Receive images from the user and call the Google Vision OCR API.
2. Charge users based on the usage.

If there are many users asking for such a service, I will try to implement it.
